app/integration_action.go

```diff
@@ -61,12 +61,11 @@ func (a *App) DoPostActionWithCookie(postId, actionId, userId, selectedOption st
 
 	// See if the post exists in the DB, if so ignore the cookie.
 	// Start all queries here for parallel execution
-	pchan := a.Srv.Store.Post().GetSingle(postId)
```

Comment:
Same here, we need to make, explicitly asynchronous to maintain the parallel behavior we had before.